### PR TITLE
add database branch password resource

### LINKED ISSUE #6: DatabaseBranchPassword
Add resource for DatabaseBranchPassword please. I may take care myself too.

### PR DESCRIPTION
**Pull request by cemo**
fixes #6

### CODE REVIEW

**Comment by cemo** [comment]
Would you please cut a release after merging it?

**Comment by cemo** [comment]
@[s1ntaxe770r](https://github.com/s1ntaxe770r/terraform-provider-planetscale/commits?author=s1ntaxe770r) would you please take a look this PR?

**Comment by s1ntaxe770r** [comment]
Oh snap , Thanks a bunch @cemo ,  looking over it now

File: planetscale/resource_branch_password.go [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+			},
+			"branch": {
+				Type:        schema.TypeString,
+				Description: "display name of your branch",
+				Optional:    true,
+				ForceNew:    true,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Description: "name of your password",
```
Comment (s1ntaxe770r) [inline planetscale/resource_branch_password.go:38]: @cemo  is this the actual value of the password or the name of the password?

**Review by s1ntaxe770r — CHANGES_REQUESTED** [review]

File: planetscale/resource_branch_password.go (hunk 1 above)
Comment (cemo) [reply planetscale/resource_branch_password.go:38]: It is actually seems on UI when you create a password. Every password is named on PS.

Comment (cemo) [reply planetscale/resource_branch_password.go:38]: If you can give a try to create a password on UI, you will immediately see.

Comment (s1ntaxe770r) [reply planetscale/resource_branch_password.go:38]: 👍🏽 Would do

Comment (cemo) [reply planetscale/resource_branch_password.go:38]: Had a chance to look? Sorry to push but we need a release :)

**Pull request merged**